puppet_fixtures.gemspec

```diff
@@ -9,10 +11,12 @@ Gem::Specification.new do |s|
   DESC
   s.authors               = ['Ewoud Kohl van Wijngaarden']
   s.files                 = ['lib/puppet_fixtures.rb', 'lib/puppet_fixtures/tasks.rb']
-  s.executables           << 'puppet-fixtures'
+  s.executables << 'puppet-fixtures'
   s.homepage              = 'https://github.com/voxpupuli/puppet_fixtures'
   s.metadata              = { 'source_code_uri' => 'https://github.com/voxpupuli/puppet_fixtures' }
   s.required_ruby_version = '>= 2.7', '< 4'
 
   s.add_dependency 'rake', '~> 13.0'
+  # 4.0 doesn't support ruby 2.7 anymore
+  s.add_development_dependency 'voxpupuli-rubocop', '~> 3.0'
```

Comment:
Please keep this in `Gemfile` under the `rubocop` group.

Reply:
doesn't the gemspec make more sense? that ensures that the gem artifact depends on it and it makes dependency tracking via rubygems easier.